gnomad_qc/v5/resources/basics.py

```diff
@@ -1,14 +1,53 @@
 """Script containing generic resources."""
 
-from gnomad.resources.resource_utils import VariantDatasetResource
+import logging
+from typing import List, Optional, Set, Union
 
-from gnomad_qc.v5.resources.constants import CURRENT_VERSION, WORKSPACE_BUCKET
+import hail as hl
+from gnomad.resources.resource_utils import (
+    VariantDatasetResource,
+    VersionedVariantDatasetResource,
+)
+from hail.utils import new_temp_file
+
+from gnomad_qc.v4.resources.basics import _split_and_filter_variant_data_for_loading
+from gnomad_qc.v5.resources.constants import (
+    AOU_GENOMIC_METRICS_PATH,
+    AOU_LOW_QUALITY_PATH,
+    CURRENT_AOU_VERSION,
+    CURRENT_VERSION,
+    GNOMAD_TMP_BUCKET,
+    WORKSPACE_BUCKET,
+)
+
+logger = logging.getLogger("basic_resources")
+logger.setLevel(logging.INFO)
 
 # v5 DRAGEN TGP test VDS.
 dragen_tgp_vds = VariantDatasetResource(
     "gs://gnomad/v5.0/testing/genomes/dragen_tgp_v5.0_test.vds"
 )
 
+_aou_genotypes = {
+    "8": VariantDatasetResource(
+        "gs://fc-aou-datasets-controlled/v8/wgs/short_read/snpindel/vds/hail.vds"
+    )
+}
+
+aou_genotypes = VersionedVariantDatasetResource(
+    CURRENT_AOU_VERSION,
+    _aou_genotypes,
+)
+
+# This test VDS was initially created to count the number of singletons in AoU and compare the results to those
+# provided in the sample QC metrics. It contains 10 selectively chosen samples based on the `singleton` metric from
```

Comment:
ah I lost my comment on this -- could you explain what "selectively chosen" means here?